docker-push.sh

```diff
@@ -19,22 +19,10 @@ then
   then
     export REACT_APP_USERS_SERVICE_URL="http://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com"
     export REACT_APP_EVENTS_SERVICE_URL="http://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com"
+    export SECRET_KEY="$STAGING_SECRET_KEY"
```

Comment:
Since our staging service is now deployed to AWS, I've added a new secret with a new SECRET_KEY to travis.